marketplace_standard_app_api/models/system.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseModel
+
+
+class GlobalSearchQuery(BaseModel):
```

Comment:
Is this the new [`QueryReply`](https://gitlab.cc-asp.fraunhofer.de/MarketPlace/platform/modules/global-search-service/-/blob/main/app/models/queryreply.py#L22)? If so, we can copy its content, right?